lib/IRGen/IRGen.cpp

```diff
@@ -79,6 +81,7 @@
 #include "llvm/Transforms/Instrumentation/SanitizerCoverage.h"
 #include "llvm/Transforms/Instrumentation/ThreadSanitizer.h"
 #include "llvm/Transforms/ObjCARC.h"
+#include "llvm/Transforms/Utils/Cloning.h"
```

Comment:
Curious what motivates this header dependency.

Reply:
Ah that's a holdover from https://github.com/apple/swift/pull/32964 when we cloned the module with the requested symbols – good spot!